### ISSUE TITLE
New term: Sudden unexpected death in epilepsy

### ISSUE THREAD

**Issue by kearneyja** [obophenotype/mammalian-phenotype-ontology#3302]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
sudden unexpected death in epilepsy

>2. Synonyms (e.g., absent spleen)
SUDEP
sudden unexplained death in epilepsy

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
Sudden, unexpected death in a person with epilepsy, with or without evidence for a seizure preceding the death, in which there is no evidence of other disease, injury, or drowning that caused the death.
PMID: 22191982

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
MP:0002064 Seizures

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
0000-0003-1726-4769

**Comment by sbello** [comment]
See related HPO issue https://github.com/obophenotype/human-phenotype-ontology/issues/6210

**Comment by sbello** [comment]
We may want to consider the existing term http://www.informatics.jax.org/vocab/mp_ontology/MP:0008028 as a pattern.

Parent term is premature death.

Need to consider if/how the term should apply to more than just humans.

**Comment by kearneyja** [comment]
MP:0008028 Premature death as a parent term makes sense. Situationally similar to pregnancy-related premature death, except the context is epilepsy rather than pregnancy.

SUDEP occurs in both humans and non-humans, including non-human primates (PMID: 31592545), dogs (PMID:17708395) and mice (e.g., PMIDs: 26272185; 23524966).

**Comment by anna-anagnostop** [comment]
Added new term:
sudden unexpected death in epilepsy MP:0031185 (child of premature death MP:0002083)
